test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -85,19 +91,35 @@ def test_dlc_version_dockerfiles(image):
                 if "example" not in dockerfile_path and f"{os.sep}{fw_version_major_minor}" in dockerfile_path:
                     dockerfiles.append(dockerfile_path)
 
+    # For the collected dockerfiles above, note the DLC major versions in each Dockerfile if python version matches
+    # the current image under test
     versions = {}
     dlc_label_regex = re.compile(r'LABEL dlc_major_version="(\d+)"')
+    python_version_regex = re.compile(r'ARG PYTHON_VERSION=(\d+.\d+)')
     for dockerfile in dockerfiles:
         with open(dockerfile, "r") as df:
+            dlc_version = None
+            python_version = None
             for line in df:
                 major_version_match = dlc_label_regex.match(line)
+                python_version_match = python_version_regex.match(line)
                 if major_version_match:
-                    versions[dockerfile] = int(major_version_match.group(1))
+                    dlc_version = int(major_version_match.group(1))
+                elif python_version_match:
+                    python_version = python_version_match.group(1).replace('.', '')
+
+            # Raise errors if dlc major version label and python version arg are not found in Dockerfile
+            if not dlc_version:
+                raise DLCMajorVersionLabelNotFound(f"Cannot find dlc_major_version label in {dockerfile}")
+            if not python_version:
+                raise DLCPythonVersionNotFound(f"Cannot find PYTHON_VERSION arg in {dockerfile}")
+            if python_version == python_major_minor_version:
+                versions[dockerfile] = dlc_version
 
     expected_versions = list(range(1, len(dockerfiles) + 1))
     actual_versions = []
 
-    for _, version in versions.items():
+    for version in versions.values():
         actual_versions.append(version)
```

Comment:
I still don't see the purpose of having this loop, rather than `actual_versions = versions.values()`.

Reply:
Not sure why it says "Outdated", but I can still see this in the code.